configmaptocrs/conversion_test.go

```diff
@@ -17,9 +17,21 @@ import (
 
 var update = flag.Bool("update", false, "update .golden files")
 
+const (
+	testConfigMapSource = false
+	testDataOnlySource  = true
+	configMapTestDir    = "./testdata/configmap"
+	dataOnlyTestDir     = "./testdata/data-only"
+)
+
 func TestGenerateResources(t *testing.T) {
+	testGenerate(t, configMapTestDir, testConfigMapSource)
```

Comment:
Can these be two different tests? It will be easier to understand what failed, to focus, etc.

Reply:
done